### PR TITLE
Update PreCAPIInstallOnBootstrap for Nutanix Provider

### PR DESCRIPTION
**Pull request by thunderboltsid**
Call provider.UpdateSecrets in PreCAPIInstallOnBootstrap.

### CODE REVIEW

File: pkg/providers/nutanix/provider.go [hunk 1]
```diff
@@ -607,8 +607,8 @@ func (p *Provider) InstallCustomProviderComponents(ctx context.Context, kubeconf
 }
 
 func (p *Provider) PreCAPIInstallOnBootstrap(ctx context.Context, cluster *types.Cluster, clusterSpec *cluster.Spec) error {
-	// TODO(nutanix): figure out if we need something else here
-	return nil
+	logger.V(0).Info("Installing secrets on bootstrap cluster")
```
Comment (abhinavmpandey08) [inline pkg/providers/nutanix/provider.go:610]: nit: you don't need to specify verbosity if it's 0
```suggestion
	logger.Info("Installing secrets on bootstrap cluster")
```

Comment (thunderboltsid) [reply pkg/providers/nutanix/provider.go]: Cool, fixed.

**Review by abhinavmpandey08 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**